### PR TITLE
TRAN-105: Add a table with Item details data and filtering

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
**Add a table with Item details data and filtering**

**Implemented solution detail:**

TRAN-105

### CODE REVIEW

File: src/stores/reducers/itemDetailReducer.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
+// import ItemActionTypes from 'stores/actions/item/itemTypes';
+// import { ItemActions, ItemState } from 'stores/types/itemType';
```
Comment (sniedzielski) [inline src/stores/reducers/itemDetailReducer.ts]: Can you remove those commented lines ?

**Review by sniedzielski — COMMENTED** [review]
one minor change to do, everything else looks fine.

File: src/stores/reducers/itemDetailReducer.ts (hunk 1 above)
Comment (KarolinaKopacz) [reply src/stores/reducers/itemDetailReducer.ts]: oh :D of course!

**Comment by sniedzielski** [comment]
@KarolinaKopacz and please fix conflicts, thx in advance!

**Comment by sniedzielski** [comment]
@KarolinaKopacz Thx for those fixes.

**Review by sniedzielski — APPROVED** [review]

**Pull request merged**